Jenkinsfile

```diff
@@ -12,7 +12,7 @@ buildMvn {
     buildDocker {
       publishMaster = true
       healthChk = false
-      healthChkCmd = 'curl -sS --fail -o /dev/null  http://localhost:8081/apidocs/ || exit 1'
+      healthChkCmd = 'wget --no-verbose --tries=1 --spider http://localhost:8081/admin/health || exit 1'
```

Comment:
did you make sure that this new url exist in this application?

Reply:
Yes, I did